NuclearData/ceNeutronData/aceDatabase/aceNeutronDatabase_class.f90

```diff
@@ -653,6 +692,9 @@ subroutine init(self, dict, ptr, silent )
       self % Ebounds(2) = min(self % Ebounds(2), self % nuclides(i) % eGrid(j))
     end do
 
+    ! Read unionised majorant flag
+    call dict % getOrDefault(self % hasMajorant, 'majorant', .false.)
```

Comment:
I would be inclined to set this to true by default. We run DT often and it's one less flag to set in that case, because we would always want the precomputed majorant (as in MG). It's also a very small price to pay if we happen to be running ST.